.jshintignore

```diff
@@ -0,0 +1,2 @@
+assets/javascripts/gridster/jquery.gridster.min.js
+assets/javascripts/jquery.knob.min.js
```

Comment:
Good point, but I think the convention for outside libraries is to place them in `vendor/assets`, right?